code/modules/ghostroles/spawner/human/admin.dm

```diff
@@ -151,10 +151,8 @@
 	name = "FIB Agent"
 	desc = "Investigate issues related to crimes under the jurisdiction of the Federal Investigations Bureau."
 
-	enabled = FALSE
 	landmark_name = "CCIAAgent"
-	req_perms = null
-	req_perms_edit = R_CCIAA
+	req_perms = R_CCIAA
 	max_count = 1
```

Comment:
Might want to remove the max_count restriction aswell if spawning itself requires perms

Reply:
I don't think there will be more than one FIB agent at a time ever?

Reply:
Currently the count can only be increased via vv, so that would leave ccia unable to spawn in more than one without help. (And we do not restrict the number of ccia agents either)

Reply:
eh fuck it